src/SpecificDissipationRateDESABLNodeSourceSuppAlg.C

```diff
@@ -0,0 +1,132 @@
+/*------------------------------------------------------------------------*/
```

Comment:
This file looks the same as the `SpecificDissipationRateSSTDESNodeSourceSuppAlg.C`. Is that true or am I missing something? If it is true, can we just use that?

Reply:
The sources are blended in this case too. if you check the last few lines, bare in mind there is not supposed to be omega solution in the ksgs region.

Reply:
Oh right I missed: `(1-fABLBlend)`. Cool